### DISCUSSION TITLE
<!DOCTYPE html> <html lang="ua"> как реализовать

### DISCUSSION THREAD

**Discussion by Aleksander2023** [maxsite/cms#499] [Q&A]
<!DOCTYPE html>
<html lang="ru">
Как увидить вместо ru -> ua
Перевожу сайт на украинский.
В mso_config раскоментировал строку $MSO->language = 'ua'; // украинский.
Админка на державній мові. А вот в !DOCTYPE html html lang="ru".
Подскажите пожалуйста как решить вопрос, где копать?
И попутно. Возможность сделать мультиязычный ua|en|sp|ru  как реализовать?
Дякую!

**Comment by maxsite** [comment]
Посмотрите в файле `custom/set_val.php`.

>> Возможность сделать мультиязычный ua|en|sp|ru как реализовать? Дякую!

Лучший вариант использовать отдельные установки системы в разных каталогах.

**Reply by Aleksander2023** [reply]
### Дякую!

**Comment by Aleksander2023** [comment]
> Лучший вариант использовать отдельные установки системы в разных каталогах.
т.е. поддомены?

**Reply by maxsite** [reply]
Технически без разницы. Зависит от того, какие хотите адреса получить: ua.site.com или site.com/ua.

**Reply by Aleksander2023** [reply]
Дякую!

**Reply by maxsite** [reply]
Будь ласка!
